packages/expo-cli/src/commands/start.js

```diff
@@ -185,7 +185,7 @@ async function configureProjectAsync(projectDir, options) {
 
   log(chalk.gray(`Starting project at ${projectDir}`));
 
-  const { exp, pkg } = await ProjectUtils.readConfigJsonAsync(projectDir, options.webOnly);
+  const { exp, pkg } = await ConfigUtils.readConfigJsonAsync(projectDir, options.webOnly);
   if (exp === null) {
```

Comment:
You can remove this `null` check now, because `readConfigJsonAsync` never returns `null`.